SU-SWS/ace-sdssgryphon
# SDSS-498: RSS feed of events for newsletter

# READY FOR REVIEW

# Summary
- Adding an RSS feed view

# Review By (Date)
- Next release!

# Urgency
- Soonish

# Steps to Test

1. Pull down this branch
2. Check for a new RSS feed of events

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- @aaroncole 
- SDSS-498

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
